statsmodels/graphics/tsaplots.py

```diff
@@ -12,7 +12,11 @@ def _prepare_data_corr_plot(x, lags, zero):
     zero = bool(zero)
     irregular = False if zero else True
     if lags is None:
-        lags = np.arange(not zero, len(x))
+        # GH 4663 - use a sensible default value
+        nobs = x.shape[0]
+        lim = min(int(np.ceil(10 * np.log10(nobs))), nobs-1)
+        lim = lim if lim <= nobs - 1 else nobs - 1
```

Comment:
isn't this redundant with the previous line?

pep-8 missing spaces in `nobs-1` on previous line

Reply:
Not needed